.pep8

```diff
@@ -1,5 +1,4 @@
 [pep8]
 ignore=E501
-max_line_length=119
-exclude=settings
-exclude=migrations
\ No newline at end of file
+max_line_length=120
```

Comment:
We use 119 [elsewhere](https://github.com/edx/course-discovery/blob/master/.pep8#L3) to keep lines under 120.